deploy-cropper.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+####################################################
+# Deploy the cropper
+#
+# Requires MCD environment variables to be in scope https://changelog.makerdao.com/releases/mainnet/active/contracts.json
+#
+# Usage: ./deploy-cropper.sh
+####################################################
+
+echo "Deploying contracts..."
+
+CROPPER=$(dapp create Cropper)
```

Comment:
Maybe adding the gas limit for each call or one that covers the maximum one?
As by default it will try to run with 200K and the contract creation ones will fail.